tests/integration/steps_multi_consumer_downtime.go

```diff
@@ -0,0 +1,408 @@
+package main
+
+// stepsDowntime tests validator jailing and slashing.
+func stepsMultiConsumerDowntimeFromConsumer(consumer1, consumer2 string) []Step {
```

Comment:
Not the biggest fan of this name haha but I can't think of a better one. Maybe you could organize the multiple consumer steps into a file called `multi-consumer-steps.go` and avoid having "multi consumer" in the name?

Not that important tho, leaving as-is could be fine

Reply:
> Seems solid 👍, it's a bit unclear why a JSON dump of test traces couldn't include steps for multiple consumers, and be tied into a variadic function, since JSON has an array type. Regardless, there's more important things to worry about and this PR is a huge improvement already

Will update in the next iteration.